requirements-dev.txt

```diff
@@ -29,6 +30,7 @@ jsonschema
 jupyter_contrib_nbextensions
 snowballstemmer==1.2.1
 addheader>=0.2.2
+highspy
```

Comment:
SCIP would work, but it makes the tests way slower. I'm solving a bunch of LPs with changes only to parameters. The Pyomo interface to HiGHS is a persistent interface, and is much faster than the interface to SCIP for that type of workflow.

Reply:
We might want to see if it makes sense to do something like @lbianchi-lbl did for SCIP then so that it does not need to be a dependency in requirements-dev.txt. If I recall, we basically have a check that sees if SCIP is available and otherwise skips the tests. We then just make sure we have it installed on the test server (via GHA setup) so the test will run on the server.

Reply:
Sounds good to me. As long as it gets installed in GHA so the tests run, I'm happy.

Reply:
Not sure if it applies, but would the other way around work? i.e. is it possible to use HiGHS instead of SCIP for our existing tests for which SCIP is currently being used?

If so, it might make sense to consider having `highspy` as a dev dependency and just use it for testing for both cases (of course, this is purely on the side of streamlining dependencies and very much solver-domain-ignorant, so feel free to add why this wouldn't work).

Reply:
I actually need both. SCIP also solves nonlinear problems, while HiGHS does not...

Reply:
Ah, that makes sense then, thanks for clarifying!